splunk-otel-android/src/main/java/com/splunk/rum/SlowRenderingDetectorImpl.java

```diff
@@ -99,24 +124,22 @@ public void add(Activity activity) {
     }
 
     @Override
-    public void stop(Activity activity) {
+    public void onActivityPaused(@NonNull Activity activity) {
         PerActivityListener listener = activities.remove(activity);
         if (listener != null) {
             activity.getWindow().removeOnFrameMetricsAvailableListener(listener);
             reportSlow(listener);
         }
     }
 
-    // the returned future is very unlikely to fail
-    @SuppressWarnings("FutureReturnValueIgnored")
     @Override
-    public void start() {
-        executorService.scheduleAtFixedRate(
-                this::reportSlowRenders,
-                pollInterval.toMillis(),
-                pollInterval.toMillis(),
-                TimeUnit.MILLISECONDS);
-    }
+    public void onActivityStopped(@NonNull Activity activity) {}
+
+    @Override
+    public void onActivitySaveInstanceState(@NonNull Activity activity, @NonNull Bundle outState) {}
```

Comment:
Really makes you wish that the interface had default nop implementations, huh? 😉

Reply:
Oh yes